### PR TITLE
Implemented count #29

### PR DESCRIPTION
**Pull request by singhashutosh96**
Many thanks for your contribution, we truly appreciate it. We will appreciate it even more, if you make sure that you can say "YES" to each point in this short checklist:

  - You made a small amount of changes (less than 100 lines, less than 10 files) YES
  - You made changes related to only one bug (create separate PRs for separate problems) YES
  - You are ready to defend your changes (there will be a code review) YES
  - You don't touch what you don't understand YES
  - You ran the build locally and it passed YES

This article will help you understand what we are looking for: http://www.yegor256.com/2015/02/09/serious-code-reviewer.html

Thank you for your contribution!

-----
[View rendered readme.md](https://github.com/singhashutosh96/enumerable4j/blob/implemented_count_#29/readme.md)

### CODE REVIEW

**Comment by singhashutosh96** [comment]
@smithros can you please help. This is my first PR . Am I doing something wrong? . On my local the build was successful. Any suggestion

**Comment by smithros** [comment]
изображение
@dgroup can you check please ?

**Comment by dgroup** [comment]
Hi @singhashutosh96,
Seems it configuration issues, don't worry we'll fix it.

File: src/main/java/io/github/dgroup/enumerable4j/Enumerable.java [hunk 1]
```diff
@@ -80,4 +80,19 @@
...
+     * @param prd The function to match each element.
+     * @return Number of elements satisfying the given function.
+     */
+    long count(Predicate<T> prd);
+
+    /**
+     * Returns size of enumerable.
+     * @return Number of elements satisfying the given function.
+     */
+    long count();
```
Comment (dgroup) [inline src/main/java/io/github/dgroup/enumerable4j/Enumerable.java:97]: this method not required as me have `size`, we need only `count(Predicate<T> prd)`.
Please remove `long count();`

**Review by dgroup — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/dgroup/enumerable4j/Enumerable.java (hunk 1 above)
Comment (singhashutosh96) [reply src/main/java/io/github/dgroup/enumerable4j/Enumerable.java]: @dgroup done

**Comment by dgroup** [comment]
@singhashutosh96 thanks a lot.
@rultor merge it.

**Comment by dgroup** [comment]
@rultor merge

**Pull request merged**